scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/ThreadedSchedulerAgent.java

```diff
@@ -64,7 +64,9 @@ public void run() {
               this.schedulerAgent.schedule(req.request(), req.writer());
             } catch (final Throwable ex) {
               ex.printStackTrace(System.err);
-              req.writer().failWith(ex);
+              req.writer().failWith(b -> b
+                  .type("UNEXPECTED_SCHEDULER_EXCEPTION")
```

Comment:
🔵 I'm wondering if it's worth having a message here - "Something went wrong while scheduling" - I could go either way.

Reply:
That's a good idea, just pushed a change to add a `.message()` here.